lib/hexpm/accounts/web_auth_request.ex

```diff
@@ -0,0 +1,26 @@
+defmodule Hexpm.Accounts.WebAuthRequest do
+  use Hexpm.Schema
+
+  @moduledoc false
+
+  # See `lib/hexpm/accounts/web_auth.ex` for an explanation of the terminology used.
+
+  @scopes ["read", "write"]
+
+  schema "requests" do
+    field :device_code, :string
+    field :user_code, :string
+    field :scope, :string
+    field :verified, :boolean
+    field :user_id, :integer
+    field :audit, :string
+  end
+
+  def changeset(request, params \\ %{}) do
+    request
+    |> cast(params, [:device_code, :user_code, :scope, :verified, :user_id, :audit])
```

Comment:
we shouldn't allow `:user_id` in the cast either. Again, cast is for user input and we don't want to accidentally ever allow someone injecting values for FKs and other internal fields.

Reply:
@wojtekmach,

Would you re-write the schema using a Many-to-one association?